src/CustomTaxonomy.php

```diff
@@ -91,6 +91,11 @@ public function create_taxonomy_metabox_markup(WP_Post $post): void
         if (-1 === $current_type) {
             // Assign default p4-pagetype for new POST.
             $planet4_default_post_type = $this->get_planet4_default_post_type();
+
+            if (!$planet4_default_post_type) {
```

Comment:
I'm not fully sure where this is used, but if we `return;` here the rest of the function won't be executed right? So the `select` component [here](https://github.com/greenpeace/planet4-master-theme/blob/b8631d4a7deddfe8efc5c81dc3fb8c8db9e8d677/src/CustomTaxonomy.php#L104-L110) won't be added?